examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -488,4 +488,4 @@
 /* End XCConfigurationList section */
 	};
 	rootObject = 97C146E61CF9000F007C117D /* Project object */;
-}
+}
```

Comment:
You've still got spurious changes to these two files.

Reply:
Fixed